api/src/main/resources/workbench-api.yaml

```diff
@@ -5434,10 +5434,6 @@ definitions:
         "$ref": "#/definitions/Workspace"
       accessLevel:
         "$ref": "#/definitions/WorkspaceAccessLevel"
-      accessedTime:
```

Comment:
Was this unused?

Reply:
yep! discussion in #workbench-techdesign